src/editor/parser/StateRender.js

```diff
@@ -112,6 +112,7 @@ class StateRender {
             default:
               break
           }
+          blockSelector += block.isLooseListItem ? '.loose-list-item' : '.tight-list-item'
```

Comment:
Pls put `.loose-list-item` and `tight-list-item` in `config.js`, It maybe write more codes, but the codes are more maintainable?

Add 'AG_' prefix to these two class name.

The class name in stylesheets are also need to change.